portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -54,6 +58,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     this.reflectionInput = request.getParameter(this.REFLECTION_PARAMETER);
     this.timestampOfComment = System.currentTimeMillis();
```

Comment:
I wouldn't use `System` for getting the current timestamp. It is pretty unreliable. Instead use [`java.time`](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html).